source/index.html.md

```diff
@@ -793,3 +793,369 @@ Account | Income Statement:Revenue:Product Revenue
+			},
+			"values": [
+				["1", "3"],
+				["20", "50"],
+				["100", "110"],
+				["10", "20"],
+				["22", "34"]
+			]
+		},
+		{
+			"range": "B7:E10",
+			"rows": {
+				"Account": ["Revenue", "COGS", "Raw COGS", "Gross Margin"]
+			},
+			"columns": {
+				"Time": ["Q1-19", "Q1-19", "Q2-19", "Q2-19"],
+				"Scenario": ["Actuals", "Budget", "Actuals", "Budget"]
+			},
+			"filters": {
+				"Department": "Consolidated Only"
+			},
+			"values": [
+				["12", "34", "99", "54"],
+				["99", "43", "76", "65"],
+				["12", "13", "23", "54"],
+				["65", "56", "34", "32"]
+			]
+		}
+	]
+}
+```
+> And returns a JSON with the same data as hitting the endpoint via GET, including the newly created Report Template
+> See [Retrieve Report Templates](#retrieve-report-templates)
+
+This endpoint allows the creation of new Report Templates in the database.
+
+You can send row, column, and filters as dimension names or IDs. The API will convert dimension names into IDs. Row/column name strings that do not match the name of a dimension are saved to preserve user-inputted data (e.g. sheet subheadings, spacing rows)
+
+Filters must only have one value. However, you can send that value as `{'top-level': 'value'}` or `{'top-level': ['value']}`. They are equivalent.
+
+
+### HTTP Request
+
+`POST https://portal.cubesoftware.com/api/v1/templates`
+
+
+## Retrieve One Report Template
+
+```python
+import requests
+
+r = requests.get('<url>', headers={
+    'Authorization': 'example-access-token'
+})
+r.json()
+```
+
+> Response is a JSON object like this:
+
+```json
+{
+    "modified_at": "2019-09-25T13:37:48.826114+00:00",
+    "created_by": {
+        "id": "1"
+    },
+    "id": "11",
+    "created_at": "2019-09-25T13:37:48.826007+00:00",
+    "name": "Retrieved Template",
+    "reports": [
+        {
+            "id": "26",
+            "sheet_range": "B1:C5",
+            "rows": {
+                "10": [ "30", "31", "32", "29" ]
+            },
+            "columns": {
+                "6": [ "9", "8" ]
+            },
+            "filters": {
+                "4": [ "5" ],
+                "1": [ "65" ]
+            },
+            "values": [
+                [ "1", "3" ],
+                [ "20", "50" ],
+                [ "100", "110" ],
+                [ "10", "20" ],
+                [ "22", "34" ]
+            ]
+        },
+        {
+            "id": "27",
+            "sheet_range": "B7:E10",
+            "rows": {
+                "1": [ "65", "66", "67", "68" ]
+            },
+            "columns": {
+                "10": [ "29", "29", "33", "33" ],
+                "6": [ "7", "8", "7", "8" ]
+            },
+            "filters": {
+                "4": [ "5" ]
+            },
+            "values": [
+                [ "12", "34", "99", "54" ],
+                [ "99", "43", "76", "65" ],
+                [ "12", "13", "23", "54" ],
+                [ "65", "56", "34", "32" ]
+            ]
+        }
+    ]
+}
```

Comment:
Maybe for this JSON we should show what data should be `POST`ed as opposed to what is returned? They are very similar, but the response structure can already be seen elsewhere (i.e. the listing endpoint).